main.go

```diff
@@ -140,17 +143,13 @@ func run(ctx context.Context) error {
 	// Start healthcheck tickers
 	healthcheck.Start(ctx)
 
-	s := server.New(cfg.BindAddr, router)
-	s.HandleOSSignals = false
-
-	s.Middleware["CollectionID"] = collectionID.CheckCookie
-	s.MiddlewareOrder = append(s.MiddlewareOrder, "CollectionID")
-
-	s.Middleware["AccessToken"] = accessToken.CheckCookieValueAndForwardWithRequestContext
-	s.MiddlewareOrder = append(s.MiddlewareOrder, "AccessToken")
+	collectionIDMiddleware := dpnethandlers.CheckHeader(dpnethandlers.CollectionID)
```

Comment:
sorry, maybe I didn't give enough info of the new generic handlers, we have:
- `func CheckHeader(key Key) func(http.Handler) http.Handler` : gets the header and stores it as a context value
- func CheckCookie(key Key) func(http.Handler) http.Handler : gets the cookie and stores it as a context value

In this case, the old handler was CheckCookie, so here we should do:
```
	collectionIDMiddleware := dpnethandlers.CheckCookie(dpnethandlers.CollectionID)
```